ui/pages/swaps/prepare-swap-page/prepare-swap-page.js

```diff
@@ -1036,6 +1036,9 @@ export default function PrepareSwapPage({
                   ? t('swapTokenVerifiedOn1SourceTitle')
                   : t('swapTokenAddedManuallyTitle')
               }
+              titleProps={{
```

Comment:
Here we are removing the static data-testid from the `BannerBase` component and utilizing `titleProps` to assign one at implementation